chart/compass/values.yaml

```diff
@@ -141,6 +141,10 @@ global:
     graphql:
       external:
         port: 3000
+    tls:
```

Comment:
This is influenced by the gateway chart. Unfortunately, I'm not sure whether this host is considered as TLS in any way. The only thing that it's doing is expecting a valid JWT token. Maybe we should drop the `tls` property since it's misleading.

Reply:
Nevermind, it's okay. It was discussed offline.